incubator/kylo/templates/job/create_elasticsearch_indicies.yaml

```
@@ -0,0 +1,24 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: create-elasticsearch-indicies
+  annotations:
+    "helm.sh/hook": "post-install"
+    "helm.sh/hook-delete-policy": "before-hook-creation"
+spec:
+  backoffLimit: 10
+  activeDeadlineSeconds: 500
+  template:
+    spec:
+      containers:
+      - name: create-elasticsearch-indicies
+        image: cfmanteiga/alpine-bash-curl-jq:latest
+        command: ["bash", "-c", "mkdir /tmp/create/; cp /etc/config/* /tmp/create/; chmod +x /tmp/create/create-kylo-indexes-es.sh; while ! curl --output /dev/null --silent --head --fail http://kylo-elasticsearch-client:9200; do sleep 1; done; ./tmp/create/create-kylo-indexes-es.sh kylo-elasticsearch-client 9200 1 1"]
```

Is this boilerplate for how it was initially? Can we break it out into separate lines and a multiline string?
```suggestion
        command:
        - "bash"
        - "-c"
        - |
          mkdir /tmp/create/
          cp /etc/config/* /tmp/create/
          chmod +x /tmp/create/create-kylo-indexes-es.sh
          while ! curl --output /dev/null --silent --head --fail http://kylo-elasticsearch-client:9200; do 
            sleep 1;
          done
          ./tmp/create/create-kylo-indexes-es.sh kylo-elasticsearch-client 9200 1 1
```

---

or better yet could this be a script stored in a configmap that just gets mounted into the job container and then the command is: `create_index.sh`?

---

I had this conversation with Jeff, but for posterity sake will restate here: we tried that and it mounts as a read-only file so those files cannot be executed from that directory and need to be copied out.